temp/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM nvcr.io/nvidia/cuda:12.5.0-base-ubuntu20.04
```

Comment:
Why do we need this?

Reply:
I pushed that by mistake. Thanks for catching. I have removed it now.